frontend/src/App.tsx

```diff
@@ -1,26 +1,1041 @@
 import React from 'react';
 import logo from './logo.svg';
 import './App.css';
+import CocktailsOnHomepage from "./compononents/CocktailsOnHomepage";
+import Header from "./compononents/Header";
+import SearchFeld from "./compononents/SearchFeld";
+import {ChangeEvent, useState} from "react";
 
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.tsx</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+export default function App() {
+
+    const cocktailsResponse = {
```

Comment:
Warum hast du hier deine ganzen Daten drin? Sollten die nicht lieber im Backend in der DB sein?